src/AntiCorruptionLayer/MatchAllMessageFilter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EventSauce\EventSourcing\AntiCorruptionLayer;
+
+use EventSauce\EventSourcing\Message;
+
+class MatchAllMessageFilter implements MessageFilter
```

Comment:
I think a better name might be `MatchAllMessageFiltersMessageFilter` since based on conventions this looks like it is saying it is the "match all" message filter. However, this is actually saying it is going to ensure the message matches all of the message filters.

This is related to something I'm going to suggest in a standalone comment.

Reply:
I've changed it to MatchAllMessageFilters, which is a little bit more concise.